doc/site/content/docs/guides/resource-excess/_index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Resource excess"
+author = "sprunk"
++++
+
+### Resource excess
+When a player generates more resources in a GameFrame than they can store, the leftovers are called **excess** (also often called overflow -- either as a noun *("I received overflow.")* or as a verb *("I overflow to my team.")*. 
+
+In some games, the overflow level can be set to something lower than 100% storage. In the example below, the "red slider" (overflow level) is set to 75% of player-storage.
```

Comment:
AFAICT engine exclusively calls it "excess" everywhere public-facing, and overwhelmingly internally as well